engineering_tools/RouteCreation_CSV2Yaml.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
If this is running in ubuntu 20.04 do we need to specify python3 ?

Reply:
Yes, that's the common practice. Also, 'python' is not available, by default:

```
$ docker run -it ros:noetic
root@ef226c5d7aa2:/# python
bash: python: command not found
```